domain/user/repository.go

```diff
@@ -19,6 +19,7 @@ func NewRepository(db infrastructure.Database, logger framework.Logger) Reposito
 	return Repository{db, logger}
 }
 
+// For AutoMigrating (used in fx.Invoke)
 func Migrate(r Repository) error {
 	r.logger.Info("[Migrating...User]")
 	if err := r.DB.AutoMigrate(&models.User{}); err != nil {
```

Comment:
Let revert these changes ! @Manjish

Reply:
will be fixed by https://github.com/wesionaryTEAM/go_clean_architecture/pull/114